src/Build/Resources/Strings.resx

```diff
@@ -1899,4 +1899,37 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
   <data name="ProxyRequestNotScheduledOnInprocNode" xml:space="preserve">
     <value>MSB4274: Disabling the inproc node leads to performance degradation when using project cache plugins that emit proxy build requests.</value>
   </data>
+  <data name="ProjectCacheBeginBuild" xml:space="preserve">
+    <value>Initializing project cache plugin.</value>
+  </data>
+  <data name="ProjectCacheEndBuild" xml:space="preserve">
+    <value>Shutting down project cache plugin.</value>
+  </data>
+  <data name="ProjectCacheQueryStartedWithTargetNames" xml:space="preserve">
+    <value>Project cache query "{0}" ({1} target(s)):</value>
```

Comment:
Totally open to suggestion for the copy text on all of these.

Reply:
```suggestion
    <value>Querying project cache for "{0}" (targets: {1}):</value>
```

?